api/WebGL2RenderingContext.json

```diff
@@ -436,10 +436,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebGL2RenderingContext/bufferSubData",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "56"
```

Comment:
OK, so the whole feature was shipped in Chrome 56 and Chrome for Android 58, and almost all of the rows in https://developer.mozilla.org/en-US/docs/Web/API/WebGL2RenderingContext#Browser_compatibility already match this. Makes sense.